src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/ExceptionRecordTest.java

```diff
@@ -38,4 +39,27 @@ public void should_upload_blob_and_create_exception_record() throws Exception {
             .pollInterval(500, TimeUnit.MILLISECONDS)
             .until(() -> Objects.equals(ProcessorEnvelopeStatusChecker.checkStatus(zipFileName), "NOTIFICATION_SENT"));
     }
+
+    @Test
+    public void should_dispatch_blob_and_create_exception_record_for_supplementary_evidence_with_ocr_classification()
+        throws Exception {
+        String zipFileName = ZipFileHelper.randomFileName();
+
+        var zipArchive = ZipFileHelper.createZipArchive(
+            singletonList("1111002.pdf"),
+            "supplementary_evidence_with_ocr_metadata.json",
+            zipFileName
+        );
+
+        await("File " + zipFileName + " should be dispatched")
+            .atMost(60, TimeUnit.SECONDS)
+            .pollInterval(500, TimeUnit.MILLISECONDS)
+            .until(() -> Objects.equals(RouterEnvelopesStatusChecker.checkStatus(zipFileName), "DISPATCHED"));
+
+        await("Exception record is created for " + zipFileName)
+            .atMost(60, TimeUnit.SECONDS)
+            .pollInterval(500, TimeUnit.MILLISECONDS)
+            .until(() -> checkEnvelope(zipFileName, "NOTIFICATION_SENT", "EXCEPTION_RECORD"));
```

Comment:
isn't envelope `COMPLETED` at this stage?

Reply:
Good point, it should be completed. Also, could we first wait just for the status and then run more assertions? Currently, we'll only see true/false, but if the test fails, we'll know very little about what went wrong.

Reply:
existing test is passing with this status.

Reply:
> existing test is passing with this status.
fixed here https://github.com/hmcts/bulk-scan-end-to-end-tests/pull/46

Reply:
changed